deploy/example/smb-provisioner/smb-server-deployment.yaml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
can you run `yamllint smb-server-deployment.yaml` to validate the yaml format first?
and also add this path `deploy/example/smb-provisioner/*.yaml` in https://github.com/kubernetes-csi/csi-driver-smb/blob/master/hack/verify-yamllint.sh#L23

Reply:
ok!